code/modules/fulton/fulton.dm

```diff
@@ -0,0 +1,153 @@
+var/list/normal_extraction_beacons = list()
+var/list/total_extraction_beacons = list()
+
+/obj/item/weapon/extraction_pack
+	name = "Fulton Recovery Pack"
+	desc = "A balloon that can be used to extract a target to a Fulton Recovery Beacon. Anything not bolted down can be moved. Link the pack to a beacon by using the pack in hand."
+	icon = 'icons/obj/fulton.dmi'
+	icon_state = "extraction_pack"
+	var/obj/machinery/extraction_point/beacon
+	var/is_syndie = 0
+	var/uses_left = 12
+
+/obj/item/weapon/extraction_pack/examine()
```

Comment:
good to know we support that now, i hated using usr but I thought it was the only way still for examine